docs/userguide/futures.rst

```diff
@@ -3,49 +3,80 @@
 Futures
 =======
 
-When a Python function is invoked, the Python interpreter waits for the function to complete execution
-and returns the results. When functions execute for a long period of time it may not be desirable to wait for completion, instead it is often preferable that the function executes asynchronously. Parsl provides such asynchronous behavior by returning a `future <https://en.wikipedia.org/wiki/Futures_and_promises>`_ in lieu of results.
-A future is essentially an object that can be used track the status of an asynchronous task so that it may, in the future, be interrogated to find the status,
+When an ordinary Python function is invoked in a Python program, the Python interpreter waits for the function to complete execution
+before proceeding to the next statement. 
+But if a function is expected to execute for a long period of time, it may be preferable not to wait for its completion but instead to proceed immediately with executing subsequent statements.
+The function can then execute concurrently with that other computation.
+
+Concurrency can be used to enhance performance when concurrent activities can execute on different cores or nodes,
+as in the following code fragment, where we may save time by executing the two function calls at the same time.
+
+   .. code-block:: python
+
+       v1 = expensive_function(1)
+       v2 = expensive_function(2)
+       result = v1 + v2
+     
```

Comment:
if this is meant to be parsl code, then v1 should be v1.result()  and likewise for v2

Reply:
I think this is meant to showing where concurrent execution can help.It's followed by parsl code in the AppFutures code sample. Might be better to clarify though.